.travis.yml

```diff
@@ -20,29 +17,38 @@ before_install:
   - git config --global user.name "Travis Deploy"
   - git config --global user.email "alongd@mit.edu"
 
+install: 
+  # set up by installing all dependencies
+  - echo $TRAVIS_OS_NAME
+  - export PYTHONPATH=$PYTHONPATH:$(pwd)
+  - echo "export PYTHONPATH=$PYTHONPATH:$(pwd)" >> ~/.bashrc
+  - conda env create -f environment.yml
+  - source activate t3_env
+  - make install-arc
+  - make install-pyrms
+  - cd ../RMG-Py
+  - export PYTHONPATH=$PYTHONPATH:$(pwd)
+  - echo "export PYTHONPATH=$PYTHONPATH:$(pwd)" >> ~/.bashrc
+  - make
+  - cd ../ARC
+  - export PYTHONPATH=$PYTHONPATH:$(pwd)
+  - echo "export PYTHONPATH=$PYTHONPATH:$(pwd)" >> ~/.bashrc
+  - cd ../T3
+
 cache: pip
 
 jobs:
   include:
-    - stage: test
-      install:
-        - echo $TRAVIS_OS_NAME
-        - export PYTHONPATH=$PYTHONPATH:$(pwd)
-        - echo "export PYTHONPATH=$PYTHONPATH:$(pwd)" >> ~/.bashrc
-        - conda env create -f environment.yml
-        - source activate t3_env
-        - make install-arc
-        - make install-pyrms
-        - cd ../RMG-Py
-        - export PYTHONPATH=$PYTHONPATH:$(pwd)
-        - make
-        - cd ../ARC
-        - export PYTHONPATH=$PYTHONPATH:$(pwd)
-        - cd ../T3
-      script:
-        - coverage run -m pytest -ra -vv
-        - coverage report
-        - codecov
+    - script:
+      - source activate t3_env
+      - coverage run -m pytest --ignore=tests/test_simulate_adapters/ -ra -vv
+      - coverage report
+      - codecov
+    - script:
+      - source activate t3_env
+      - travis_wait 45 coverage run -m pytest tests/test_simulate_adapters/ -vv
```

Comment:
nice!
will we get two coverage reports?
why wait 45 (seconds?)?

Reply:
I don't think so? RMG-Py runs multiple tests in [parallel](https://travis-ci.org/github/ReactionMechanismGenerator/RMG-tests/builds/752092205) but only generates one coverage [report](https://github.com/ReactionMechanismGenerator/RMG-Py/runs/1625449533). 

At the time of this PR, SA was slow in pyrms so I made travis wait to avoid timing out. I think Matt and/or Hao-Wei have fixed this bug in RMS since then, but I'd prefer to make any pyrms changes in a future PR because it was very finicky for this PR.